dcicutils/env_utils.py

```diff
@@ -122,6 +122,36 @@
 
 ]
 
+BEANSTALK_DEV_DATA_SETS = {
+
+    'fourfront-hotseat': 'prod',
+    'fourfront-mastertest': 'test',
+    'fourfront-webdev': 'prod',
```

Comment:
OK, we talked through this. I see this on my machine:
```
$ show_deployed_file fourfront-webdev production.ini | grep load_test_data
load_test_data = encoded.loadxl:load_prod_data
```
So that's the value in `production.ini` on the deployed machine. This value is not telling you that it loads prod data, it's telling you that it doesn't load additional test data, so the naming is confusing. I wrote ticket [C4-139](https://hms-dbmi.atlassian.net/browse/C4-139) to track this.